test/test_llama2c_tokenizer.cpp

```diff
@@ -6,34 +6,19 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-#ifdef TOKENIZERS_FB_BUCK
-#include <TestResourceUtils/TestResourceUtils.h>
-#endif
 #include <gtest/gtest.h>
 #include <pytorch/tokenizers/llama2c_tokenizer.h>
 
 using namespace ::testing;
 
 namespace tokenizers {
 
-namespace {
-// Test case based on llama2.c tokenizer
-static inline std::string _get_resource_path(const std::string& name) {
```

Comment:
I guess you can change the logic of this function and keep the call sites intact?